extension/encoding/avrologencodingextension/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+type: avro_log_encoding
+scope_name: otelcol/avrologencoding
+
+status:
+  class: extension
+  stability:
+    development: [extension]
+  distributions: []
+  codeowners:
+    active: []
```

Comment:
@thmshmm could you please put yourself as the code owner to help maintain the extension it going forward?